frontend/src/app/organization/organization-editor/organization-editor.component.html

```diff
@@ -53,23 +53,29 @@
         appearance="outline"
         color="accent">
         <mat-label>Short Description</mat-label>
-        <input
+        <textarea
           matInput
           placeholder="Enter short description here."
           formControlName="short_description"
-          name="short_description" />
+          name="short_description"></textarea>
       </mat-form-field>
       <!-- Long Description Field -->
       <mat-form-field
+        class="long-description"
         hintLabel="Max 2000 characters"
         appearance="outline"
         color="accent">
         <mat-label>Long Description</mat-label>
-        <input
+        <textarea
+          matInput
+          placeholder="Enter long description here."
+          formControlName="long_description"
+          name="long_description"></textarea>
+        <!-- <input

```

Comment:
I might argue that keeping short and long description fields full width would be better here, since they are text areas! Both fields next to each other might be a bit cramped and hard to edit long text in.

Reply:
Definitely agree with this!